src/base/bittorrent/session.h

```diff
@@ -122,6 +122,14 @@ namespace BitTorrent
         };
         Q_ENUM_NS(ChokingAlgorithm)
 
+        enum class DiskIOType : int
+        {
+            Default = 0,
+            Posix = 1,
+            MMap = 2
```

Comment:
I think it should be:
```suggestion
            Mmap = 2
```

Reply:
Please have an agreement on the naming, it was Mmap initially, then changed to MMap.
I prefer to stay with MMap, since it's M(emory)-Map.

Reply:
> I prefer to stay with MMap, since it's M(emory)-Map.

Exactly 👍

Reply:
And Posix should be POSIX.

Reply:
We should use correctly written abbreviations in user visible strings as a mandatory and in the code as much as possible.